spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -265,7 +265,7 @@ Embedded objects can be stored in the values of embedded Maps.
 Collections of embedded objects can also be stored as the values of embedded Maps.
 
 
-==== Relationships
+==== Implicit Ancestor/Descendant Relationships
```

Comment:
I'm not sure about the use of the term "implicit" here. Maybe just "Ancestor-descendant Relationships"?